pkg/combination/combination.go

```diff
@@ -69,10 +69,6 @@ func (cs *streamImp) Next(ctx context.Context) (map[string]string, error) {
 		}
 	}
 
-	if cs.solveAhead && !cs.solved {
-		cs.solve()
-	}
```

Comment:
Any reason this was removed? The linter complained about it?

Reply:
It was some left over code I forgot to remove in a previous PR it looks like and the linter caught the error wasn't checked. If you look at the line above it is doing the same thing but checking the error.